File: WebApp/app/model/Customer.js
Ext.define('WebApp.model.Customer', {
    extend: 'Ext.data.Model',
	idProperty : 'CustomerId', 
    fields: [
		{name : 'Name'}, 
		{name : 'Code'}, 
		{name : 'Addres'},
		{name : 'Discount'}
	],
	hasMany : {
        model : 'Order',
        associationKey : 'orders'
    }
});

File: WebApp/Models/Customer.cs
namespace WebApp.Models
{
    using Newtonsoft.Json;
    using System;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Customer")]
    public partial class Customer
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Customer()
        {
            Order = new HashSet<Order>();
        }

        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public Guid CustomerId { get; set; }

        [Required]
        [StringLength(255)]
        public string Name { get; set; }

        [DefaultValue("0000-2020")]
        [StringLength(14)]
        public string Code { get; set; }

        [StringLength(255)]
        public string Address { get; set; }

        public int? Discount { get; set; }

        [JsonIgnore]
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<Order> Order { get; set; }
    }
}
